src/PrestaShopBundle/Resources/views/Admin/Sell/Catalog/Product/Blocks/tabs.html.twig

```diff
@@ -23,6 +23,9 @@
  * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
  *#}
 {% set isBasicTabValid = productForm.basic.description.vars.valid and productForm.basic.description_short.vars.valid %}
+{% set isBasicTabValid = isBasicTabValid and productForm.manufacturer.vars.valid %}
```

Comment:
Impossible to have multiple `and` in Twig?

Suggestion not blocking: extend Twig, add a little twig function `isTrueIfAllAreTrued()`

Or is it overengineered ? 😄

Reply:
Oh no it's possible it's just to avoid having a line with 12000 columns